core/src/main/java/com/radixdlt/sync/LocalSyncService.java

```diff
@@ -568,10 +548,7 @@ private <T extends SyncState> T updateSyncTargetDiffCounter(T syncState) {
       this.metrics
           .sync()
           .targetStateVersion()
-          .set(
-              Math.max(
-                  syncingState.getTargetHeader().getStateVersion(),
-                  syncingState.getTargetHeader().getAccumulatorState().getStateVersion()));
```

Comment:
_(note from author)_

Does anyone know what the `.max(X, X)` here was really doing? Should it be doing something else?